config/config/sc-config.yaml

```diff
@@ -68,54 +68,78 @@ user:
 
 harbor:
   enabled: true
+  # The tolerations, affinity, and nodeSelector are applied to all harbor pods.
   tolerations: []
   affinity: {}
   nodeSelector: {}
   chartmuseum:
     persistentVolumeClaim:
       size: 5Gi
+    resources:
+      requests:
+        memory: 16Mi
+        cpu: 5m
   core:
     resources:
       requests:
-        memory: 256Mi
-        cpu: 100m
-      limits:
-        memory: 512Mi
-        cpu: 200m
+        memory: 64Mi
+        cpu: 10m
   database:
     persistentVolumeClaim:
       size: 1Gi
+    resources:
+      requests:
+        memory: 512Mi
+        cpu: 250m
   jobservice:
     persistentVolumeClaim:
       size: 1Gi
     resources:
       requests:
-        memory: 256Mi
-        cpu: 100m
-      limits:
-        memory: 512Mi
-        cpu: 200m
+        memory: 64Mi
+        cpu: 10m
   registry:
     persistentVolumeClaim:
       size: 5Gi
     resources:
       requests:
-        memory: 256Mi
-        cpu: 100m
-      limits:
-        memory: 512Mi
-        cpu: 200m
+        memory: 64Mi
+        cpu: 10m
+    controller:
+      resources:
+        requests:
+          memory: 16Mi
+          cpu: 1m
   redis:
     persistentVolumeClaim:
       size: 1Gi
+    resources:
+      requests:
+        memory: 32Mi
+        cpu: 10m
   notary:
     resources:
       requests:
-        memory: 256Mi
-        cpu: 100m
-      limits:
-        memory: 512Mi
+        memory: 16Mi
+        cpu: 5m
+  notarySigner:
+    resources:
+      requests:
+        memory: 16Mi
+        cpu: 5m
+  portal:
+    resources:
+      requests:
+        memory: 16Mi
+        cpu: 5m
+  trivy:
+    persistentVolumeClaim:
+      size: 5Gi
+    resources:
+      requests:
         cpu: 200m
+        memory: 512Mi
+      limits: null #the chart has default limits
```

Comment:
So does this make it so that the default limits isn't used?

Reply:
Yes

Reply:
And why do we not want to use the default limits?

Reply:
I mostly added this to be in line with the other pods for harbor. There has been some discussions on if you should have limits, at least for CPU. 
But if you want us to have the limits then I'm fine with that as well.

Reply:
Yes for CPU I can understand. But I guess for memory it could make sense. Especially since the upstream chart has it for this but not for the others, then it might be there for a reason.
But I'm also okey with leaving it as is.

Reply:
Tried to do some digging in the history of the chart but I could not find any reason why the limits/requests were at those values. 
I guess it could be cleaner just use the limits from the chart. Though they would be slightly "hidden" when looking at these values, since they are just propagated from the char.